pygfx/renderers/wgpu/_blender.py

```diff
@@ -316,14 +316,14 @@ def get_color_attachments(self, blender, clear_color):
                 "view": blender.accum_view,
                 "resolve_target": None,
                 "clear_value": accum_clear_value,
-                "load_op": wgpu.logger.clear,
+                "load_op": wgpu.LoadOp.clear,
                 "store_op": wgpu.StoreOp.store,
             },
             {
                 "view": blender.reveal_view,
                 "resolve_target": None,
                 "clear_value": reveal_clear_value,
-                "load_op": wgpu.logger.clear,
+                "load_op": wgpu.LoadOp.clear,
```

Comment:
These two "spelling errors" should be merged into master asap, the other change (alpha) is probably going to need to some input from @almarklein

Reply:
If you can split the PR we can merge this part immediately:)

Reply:
See #314